kivy/config.py

```diff
@@ -709,6 +709,7 @@ def name(self, value):
             # from 1.0.9, don't activate mactouch by default, or app are
             # unusable.
             if platform == 'win':
+                Config.setdefault('input', 'wm_pointer', 'wm_pointer')
```

Comment:
This needs to be added and documented with the other config variables in kivy/__init__.py (I believe).